from app.exceptions.api import APIError


class NotFoundError(APIError):
    pass
